app/controllers/application_controller.rb

```diff
@@ -142,6 +143,19 @@ def current_user_redirect_path
     current_user.getting_started? ? getting_started_path : stream_path
   end
 
+  def gon_set_appconfig
+    gon.push({
+      appConfig: {
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.